docs/acp/user/intro-to-acp/acp-installation.md

```diff
@@ -0,0 +1,485 @@
+---
+title: App Composition Platform Installation
+description: Learn how to install the App Orchestration Platform.
+template: concept-topic-template
+redirect_from:
+    - /docs/aop/user/intro-to-acp/acp-installation.html
+---
+
+The App Composition Platform (ACP) lets Spryker Cloud customers connect, configure, and use the available third-party services or apps, in their application with a click of a button, without development efforts from their side.
```

Comment:
"without development efforts from their side" is not true TBH, I think what we can say right now is that the effort is low but still it is there.

Reply:
"3rd party service" refers to all apps which are basically a proxy for 3rd party services (such as payment, user reviews, etc..) vs "standalone" apps. Since that is not the case I changed it up a little to make clear "3rd party services via apps".

I softened the "no code" and added "no to low", which is for example true for UserCentrics